src/components/Button/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import { FiDownload } from "react-icons/fi";
+import "./button.css";
+import { ButtonProps } from "./types";
+
+const Button = ({ type = "primary", disabled, children }: ButtonProps) => {
+  const className = ` button ${type}`;
+  const addClickHandler = () => {};
```

Comment:
click method should be a callback method because we need to write login in the consumed component.